public/index.html

```diff
@@ -10,34 +10,13 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
+ 
```

Comment:
Is this file an "error handling pathway" for users who don't have JavaScript enabled?

Reply:
To me, this is just some instruction by React but we don't need them. This file is what display our entire React application and translated into what is called the DOM.